hash/ibm5170.xml

```diff
@@ -9659,7 +9659,7 @@ Missing [modem] features, assume requiring a connection to a Reuters device such
 		</part>
 	</software>
 
-	<software name="aopacex">
+	<software name="aopacex" cloneof="aopac">
 		<description>Aces of the Pacific Expansion Disk - WWII: 1946</description>
```

Comment:
Again: it's all over the place in `hash`, I didn't make this rule but I'm inclined to agree over the fact that "you need a parent in order to make the clone to work".

Reply:
The thing is, “you need a parent to make the clone work” isn’t how clone systems work at all.  You only need the ROMs that are actually part of the clone itself for it to work.  It’s just that MAME searches the parent as a convenience because there are often ROMs in common.

We could probably do with some other markup for expansions/updates, but it’ll get complicated with things like e.g. a program port for Mac that only contains the actual application, and requires the original DOS disks to install assets.